### PR TITLE
 Updated LetsExpr and LetsStmt, added tests

### PR DESCRIPTION
**Pull request by MichaelS11**
Please note this does change how Run output value behavior for let statements when they have multiple values, it will only return the last value set.

### CODE REVIEW

File: vm/vmExpr.go [hunk 1]
```diff
@@ -541,37 +541,28 @@ func invokeExpr(expr ast.Expr, env *Env) (reflect.Value, error) {
...
+			if v.IsValid() && v.Kind() == reflect.Interface && !v.IsNil() {
 				v = v.Elem()
 			}
 			_, err = invokeLetExpr(lhs, v, env)
 			if err != nil {
 				return NilValue, NewError(lhs, err)
 			}
 		}
-		return rvs, nil
+		return rvs[len(rvs)-1], nil
```
Comment (mattn) [inline vm/vmExpr.go]: what this mean?

File: vm/vmStmt.go [hunk 2]
```diff
@@ -43,68 +43,56 @@ func RunSingleStmt(stmt ast.Stmt, env *Env) (reflect.Value, error) {
...
-			if i < len(rvs) {
-				env.Define(name, rvs[i])
-				result = append(result, rvs[i].Interface())
+			if i >= len(rvs) {
+				break
 			}
+			env.Define(name, rvs[i])
 		}
-		return reflect.ValueOf(result), nil
+		return rvs[len(rvs)-1], nil
```
Comment (mattn) [inline vm/vmStmt.go]: ditto

**Comment by MichaelS11** [comment]
This PR changes how multiple variable set expressions return run output. For example:
```
a, b = 1, 2
```
Run output before PR:
```
[] interface{}{int64(1), int64(2)}
```
Run output after PR:
```
int64(1)
```

This goes for multiple variable var statements as well.

**Comment by MichaelS11** [comment]
Updated interrupt, changed to be a flag that can be set and unset for environment group.

**Comment by mattn** [comment]
Ah, I understand what this mean.

**Comment by mattn** [comment]
Thank you

**Pull request merged**